bin/crew

```diff
@@ -471,7 +470,7 @@ def download
 
   if !url
     abort "No precompiled binary or source is available for #{@device[:architecture]}.".lightred
-  elsif url.casecmp?('SKIP') || (@pkg.no_source_build? || @pkg.gem_compile_needed?)
+  elsif url.casecmp?('SKIP') || ((@pkg.no_source_build? || @pkg.gem_compile_needed?) && caller.grep(/download_command/).empty?)
```

Comment:
How about this?
```ruby
caller_locations(1,1)[0].label == 'download_command'
```

Reply:
Does it work?

Reply:
That doesn't actually work the next time I use it, because that just outputs `Dir.chdir`...